media/js/app/map.js

```diff
@@ -101,11 +101,8 @@
                 }
             }
 
-            var iconURL = self.model.get('video').length < 1 ?
-                ('https://teachdentistry.org/site_media/img/' +
-                 'icon-pointer-indigo.png') :
-                    ('https://teachdentistry.org/site_media/img/' +
-                     'icon-pointer-red.png');
+            var iconURL = this.staticURL + self.model.get('video').length < 1 ?
+                'icon-pointer-indigo.png' : 'icon-pointer-red.png';
```

Comment:
I think you need parentheses around `self.model.get('video').length < 1 ? 'icon-pointer-indigo.png' : 'icon-pointer-red.png'`.

I just tried evaluating this expression in javascript: `'a' + 1 < 2 ? 'b' : 'c';` and it yields "c" instead of "ab"